options/options.c

```diff
@@ -715,6 +715,10 @@ const m_option_t mp_opts[] = {
                       0, INT_MAX, ({"auto", -1})),
 #endif
 
+#if HAVE_VAAPI_HWACCEL
+    OPT_STRING("vaapi-device", vaapi_device, M_OPT_FILE),
```

Comment:
since there appears to be some demand for this on other platforms / hwdec apis, it might be better to have  a more generic options similar to `--audio-device.`